packages/v2/gui/src/sak/meldinger/TredjepartsmottakerInput.tsx

```diff
@@ -54,12 +54,12 @@ const TredjepartsmottakerInput = ({
   const lookupAborterRef = useRef<AbortController | undefined>(undefined);
 
   const handleInputOnChange = async (orgnr: string) => {
-    try {
-      const prevTredjepartsmottaker = tredjepartsmottaker;
-      lookupAborterRef.current?.abort(new RequestIntentionallyAborted());
-      lookupAborterRef.current = new AbortController();
-      const newTredjepartsmottaker =
-        orgnr.length > 0 ? await api.getBrevMottakerinfoEreg(orgnr, lookupAborterRef.current?.signal) : undefined;
+    const prevTredjepartsmottaker = tredjepartsmottaker;
+    lookupAborterRef.current?.abort();
+    lookupAborterRef.current = new AbortController();
```

Comment:
Henger ikke helt med her. Hvorfor kjøres det `abort` i linja over og så settes en ny `AbortController` her?

Reply:
Det er fordi lookupAborterRef held (muligens) på abort signal for request som vart starta ved forrige tastetrykk i input boks.

Så viss eit tastetrykk kjem inn før request som vart trigga på forrige tastetrykk er ferdig, så blir forrige request avbrutt.

Så blir ny abortcontroller for noverande tastetrykk oppretta.